### PR TITLE
Lqnguyen branch1

### PR DESCRIPTION
**Pull request by lqnguyen**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue) 

## Type of change

Please delete options that are not relevant. Add a 'X' to the one that is applicable. 

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Sample Migration (Moving sample from old repository after completing checklist established)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Command Line
- [ ] oneapi-cli
- [ ] Visual Studio
- [ ] Eclipse IDE
- [ ] VSCode

# Checklist for Moving samples:
Links and Details can be found in the samples WG Teams Files. 

- [ ] Review sample design with domain reviewers https://github.com/oneapi-src/oneAPI-samples/wiki/Reviewers-and-Domain-Experts 
- [ ] Implement coding guidelines and ensure code quality.
- [ ] Adhere to sample.json specification. https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-specification
- [ ] Run jsonlint on sample.json to verify json syntax. www.jsonlint.com
- [ ] Adhere to readme template 
- [ ] Ensure/create CI test configurations for sample (ciTests field) https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-ci-test-object
- [ ] Enforce format via clang-format config file
- [ ] Review DPC++ code with Paul Peterseon. (GitHub User: pmpeter1)
- [ ] Review readme with Tom Lenth or Joe Oster. (GitHub User: JoeOster)
- [ ] Tested using Dev Cloud when applicable
- [ ] Implement fixes for ONSAM Jiras
- [ ] If you have new dependencies/binaries, inform Samples Project Manager Swapna R Dontharaju (@srdontha) or @JoeOster

### CODE REVIEW

**Review by pmpeter1 — CHANGES_REQUESTED** [review]
If you have "using std" in the code you should not also have std:: unless it's to resolve an ambiguity.

It works best to design your kernels inside-out.   I.e. figure out how to represent the inner loop best, and use the most obvious names (should not need _acc suffixes), then name your buffers, then name your data objects.   

The breaking of strings at likes 132-135 should be fixed to be more natural.

At 208, this is not Kernel time this is more elapsed time.  It includes data transfer cost and other overhead not just kernel execution time on the device.

File: DirectProgramming/DPC++/ParallelPatterns/PrefixSum/src/PrefixSum.cpp [hunk 1]
```diff
@@ -37,69 +39,64 @@
...
-using namespace std; 
+using namespace std;
 
-void Show(int a[], int arraysize) 
-{ 
-  for (int i = 0; i < arraysize; ++i) 
-  {
+void Show(int a[], int arraysize) {
+  for (int i = 0; i < arraysize; ++i) {
     std::cout << a[i] << " ";
```
Comment (JoeOster) [inline DirectProgramming/DPC++/ParallelPatterns/PrefixSum/src/PrefixSum.cpp:46]: redundant std:: as you are already using the std namespace

**Review by JoeOster — CHANGES_REQUESTED** [review]

**Comment by lqnguyen** [comment]
Source file has been updated based on comments. Please review.

**Review by JoeOster — CHANGES_REQUESTED** [review]
std:: is not necessary when using the std namespace unless it is to avoid confusion, please remove the extra

**Review by JoeOster — CHANGES_REQUESTED** [review]
std:: is not necessary when using the std namespace unless it is to avoid confusion, please remove the extra

**Review by pmpeter1 — CHANGES_REQUESTED** [review]
The exception handler should be from dpc_common.hpp.

The Usage() function should restructure the strings to look better when formatted with clang-format.   Clean up where line-breaks are made by the formatter.

**Review by pmpeter1 — APPROVED** [review]

**Review by JoeOster — APPROVED** [review]

**Pull request merged**